pom.xml

```diff
@@ -1054,7 +1052,7 @@
                 <module>spring-5-webflux-2</module>
                 <module>spring-activiti</module>
                 <module>spring-batch-2</module>
-                <module>spring-caching-2</module>
+				<module>spring-boot-modules/spring-caching-2</module>
```

Comment:
spring-boot-modules is already included in the pom, there is no need to include the submodule explicitly in the root pom.

Reply:
I saw that spring-caching-2 needs to run with jdk9 and above. That is why I left here and I added in the sub module style and to change that location.